### PR TITLE
Avatar component added

### PR DESCRIPTION
**Pull request by rkapoor10**
Please ignore other files and give reviews on avatar component i.e added

### CODE REVIEW

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -65,18 +65,101 @@ <h2>Avatar - Profile</h2>
...
+                            <p class="txt-s padb-1">Could be used in place of display profile based on gender</p>
+                            <div class="flex-icons display-container">
+                                <div class=" ">
+                                    <img class="avatar size-XL" src="/components/images/male.svg" alt="avatar" srcset="">
+                                </div>
+                                <div>
+                                    <img class="avatar size-L " src="/components/images/female.svg" alt="avatar" srcset="">
+                                </div>
+                                <div>
+                                    <img class="avatar size-M " src="/components/images/kidspic.jpg" alt="avatar" srcset="">
```
Comment (tushartiwari7) [inline components/avatar/avatar.html:148]: Try implementing lazy loading on these images

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -7,6 +7,22 @@
...
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+    color: var(--lightgray-color);
+}
+
+.square-avatar{
+    display: flex;
+    justify-content: center;
```
Comment (tushartiwari7) [inline components/avatar/avatar.css:21]: try making a different .flex class and reuse it whenever needed.

**Comment by AmishMishra11** [comment]
looks great you can go ahead and merge it.

**Comment by tushartiwari7** [comment]
Else all looks great! nice work.

**Review by tushartiwari7 — COMMENTED** [review]
Make this suggested changes

File: components/avatar/avatar.css (hunk 2 above)
Comment (rkapoor10) [reply components/avatar/avatar.css:21]: made the changes...thanks

File: components/avatar/avatar.html (hunk 1 above)
Comment (rkapoor10) [reply components/avatar/avatar.html]: made the changes thanks

**Pull request merged**